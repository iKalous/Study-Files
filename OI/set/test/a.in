2378 2347