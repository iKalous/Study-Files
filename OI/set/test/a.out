4725
Hellow world
